charts/ccd-message-publisher/values.aat.template.yaml

```diff
@@ -1,13 +1,3 @@
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
-  environment:
-    DATA_STORE_DB_NAME: ccd_data_store
-    DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-v11-aat.postgres.database.azure.com
```

Comment:
you still need to keep `DATA_STORE_DB_HOST` or change values.yaml to point v11 prefix